src/main/java/bio/terra/cli/command/user/Invite.java

```diff
@@ -17,20 +19,27 @@ public class Invite extends BaseCommand {
 
   @CommandLine.Option(
       names = "--enable-spend",
-      defaultValue = "false",
-      description = "Also enable the email as a user on the default spend profile.")
-  private boolean enableSpend;
+      arity = "0..1",
```

Comment:
Why did this (lines 20-25) change? I find old code easier to read

Reply:
Wanted to keep the argument structure the same, so `--enable-spend` can still be used as a flag OR you can specify a profile to go with it like `--enable-spend=test-spend`. 

Is there anything that jumps out as less readable I should address?

Reply:
I prefer to leave --enable-spend as a boolean; it makes the code easier to understand. If --enable-spend is set, then --spend-profile will be used. (If --spend-profile isn't set on command-line, default of wm-default-spend-profile will be used.)

(@cbookg another thing we could do is delete --enable-spend option and always add to spend profile. In practice, we always set --enable-spend.)

Reply:
fair enough. kept the --enable-spend flag for now but agree it should just be the default so can remove if needed